consensus/scp/tests/mock_network/cyclic_topology.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+// Ring style network topologies.
+
+// We allow dead code because not all integration tests use all of the common code.
+// https://github.com/rust-lang/rust/issues/46379
+#![allow(dead_code)]
```

Comment:
This isn't necessary (or shouldn't be), see https://github.com/brandonkal/inkjet/commit/4eb72a30345d6cb1838822d5df854430ff92f7d9 for an example of how to fix this.

Reply:
so this fixes it in the other direction I guess, by making all the tests use all the code? I feel like both of these fixes hide real dead code issues and are not ideal... I would want to leave the link to the tracking issue...